src/app/components/PortraitVideoCarousel/PortraitCarouselNavigation/index.tsx

```diff
@@ -0,0 +1,81 @@
+/** @jsx jsx */
+import { useCallback, useContext, useEffect, useState } from 'react';
+import { jsx } from '@emotion/react';
+import { NavigationButtonsProp } from '#app/models/types/portraitVideo';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import styles, { PROMO_ITEM_WIDTH } from './index.styles';
+import { LeftChevron, RightChevron } from '../../icons';
+
+const defaultTranslations = {
+  scrollLeftAria: 'Scroll left',
+  scrollRightAria: 'Scroll right',
+};
+
+export default ({ scrollPaneRef }: NavigationButtonsProp) => {
+  const { dir } = useContext(ServiceContext);
+  const [canScrollLeft, setCanScrollLeft] = useState(false);
+  const [canScrollRight, setCanScrollRight] = useState(false);
+  const {
+    translations: { carousel = defaultTranslations },
+  } = useContext(ServiceContext);
+
+  const { scrollLeftAria, scrollRightAria } = carousel;
+
+  const checkScrollButtons = useCallback(() => {
+    if (!scrollPaneRef.current) return;
+    const { scrollLeft, scrollWidth, clientWidth } = scrollPaneRef.current;
+    setCanScrollLeft(scrollLeft > 0);
+    setCanScrollRight(scrollLeft + clientWidth < scrollWidth);
+  }, [scrollPaneRef]);
+
+  const scroll = (buttonTriggered: 'left' | 'right') => {
```

Comment:
Do we want to use a type here? `ScrollDirection` (and chuck it into the types file?)